journals/apps/journals/forms.py

```diff
@@ -33,3 +32,31 @@ class UsersJournalAccessForm(forms.Form):
         label=_("Users' CSV file"),
         help_text=_(".csv file with single column consists of only usernames"),
     )
+
+
+class JournalEditForm(forms.ModelForm):
+    """
+        Journal Edit form for wagtail CMS
+    """
+    status = forms.BooleanField(label=_('Status'), required=False)
```

Comment:
Couple of comments about Status field:
- should probably be a drop down to match what is in discovery service django-admin for Journals model, the options are "Active", "Inactive", "Retired"
- Add help text, same as in discovery django-admin: "Used to determine whether journal is marketed or not."

Reply:
@bill-filler 
`publish_journal` only take two kind of parameters , i.e: `--publish` '--no-publish'. To support this type of option `Retired` we need to change command too and  add a new param that will receive status string.